### PR TITLE
Interactors styles

### PR DESCRIPTION
**Pull request by jorgelainfiesta**
## Motivation

We want to give the docs Frontside's styles.

## Approach

I focused on creating styles for the pages related to Interactors. The rest of the landing pages are still pending. 

I rephrased and reorganized the Interactors landing page. Please let me know what you think. 

[Link to preview](https://deploy-preview-721--bigtest.netlify.app/interactors)

### CODE REVIEW

**Review by cowboyd — APPROVED** [review]
So good! I've got nothing to add.

File: website/docs/src/pages/interactors.js [hunk 1]
```diff
@@ -32,146 +32,144 @@ import styles from './styles.module.css';
...
-          <div className={styles.buttons}>
+      description="Test your app as a person uses it">
+      <header>
+        <div className={clsx("container", styles.heroContainer)}>
+          <div className={styles.heroText}>
+            <h1 className={styles.heroHeading}>
+              Test your app as real people use it
+            </h1>
+            <p className={styles.heroSubheader}>
+              Interactors make it easy to test UIs at scale while keeping accesibility at the core.
```
Comment (jenweber) [inline website/docs/src/pages/interactors.js]:
```suggestion
              Interactors make it easy to test UIs at scale while keeping accessibility at the core.
```
Spellcheck

**Review by jenweber — APPROVED** [review]
Gorgeous! The color palette passes a Lighthouse audit for accessibility and the new language to hype up Interactors is awesome. I caught one spelling typo but didn't notice any other issues.

I just want to make sure you know that you can ditch the "why use interactors" section or rework it entirely. I'm not attached to it.

**Comment by jorgelainfiesta** [comment]
> I just want to make sure you know that you can ditch the "why use interactors" section or rework it entirely. I'm not attached to it.

Thanks! The content of that section is great and speaks directly to the problem, if somebody scrolled so far they probably are up for reading something longer so I think it fits in nicely. We might repurpose/reformat it in a later iteration if we see fit.

**Pull request merged**